src/grpc/mod.rs

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2021 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use futures::channel::oneshot;
+use futures::executor::block_on;
+use futures::prelude::*;
+use grpcio::{Environment, ServerBuilder};
```

Comment:
Any particular reason to choose grpcio over tonic?

Reply:
AA does not need to use the additional functions provided by tonic, only using grpcio is enough, which can reduce AA footprint. On the other hand, the current grpc version is a transitional scheme, which will be replaced with lighter ttrpc along with ocicrypt-rs.